config/initializers/spree.rb

```diff
@@ -18,6 +18,9 @@
   config.shipping_instructions = true
   config.address_requires_state = true
 
+  # Set allow_backorders to false by default in test environment
+  config.allow_backorders = false if Rails.env == 'test'
```

Comment:
As you suggest @Matt-Yorkley, depending on the result of the discourse discussion, we could potentially just set this to `false` for all environments. My understanding is that the OFN is not really set up to handle backorders properly anyway.